### PR TITLE
DEPLOY-656: Remove nginx server header

### PR DESCRIPTION
**Pull request by sergiuv2020**

### CODE REVIEW

File: helm/alfresco-infrastructure/values.yaml [hunk 1]
```diff
@@ -7,9 +7,6 @@ persistence:
     path: "/"
   baseSize: 20Gi
 
-activemq:
```
Comment (rgauss) [inline helm/alfresco-infrastructure/values.yaml:10]: Is this intentional?

File: helm/alfresco-infrastructure/values.yaml [hunk 2]
```diff
@@ -18,6 +15,4 @@ nginx-ingress:
       enabled: true  
     config:
       ssl-redirect: "false"
-
-alfresco-identity-service:
```
Comment (rgauss) [inline helm/alfresco-infrastructure/values.yaml:22]: Is this intentional?

**Review by rgauss — CHANGES_REQUESTED** [review]

File: helm/alfresco-infrastructure/values.yaml (hunk 1 above)
Comment (sergiuv2020) [reply helm/alfresco-infrastructure/values.yaml:10]: yes, because they don't do anything.

File: helm/alfresco-infrastructure/values.yaml (hunk 2 above)
Comment (sergiuv2020) [reply helm/alfresco-infrastructure/values.yaml:22]: This also has no effect on anything.

**Comment by cristina-sirbu** [comment]
I understand that removing  activemq.enabled and alfresco-identity-service.enabled doesn't have any impact but I think it's worth keeping them because people that want to customize the Infrastructure chart might look just at the values file. Not everyone is looking in every single file of the chart or documentation. Just my opinion.

**Comment by rgauss** [comment]
> I understand that removing activemq.enabled and alfresco-identity-service.enabled doesn't have any impact but I think it's worth keeping them because people that want to customize the Infrastructure chart might look just at the values file. Not everyone is looking in every single file of the chart or documentation. Just my opinion.

Agreed.

**Comment by kconkas** [comment]
Just my 5c: I find the inclusion of `activemq.enabled` and `alfresco-identity-service.enabled` in `values.yaml` (despite the fact they don't control anything but simply "are there") rather misleading. Take a look at AUTH-251 for a recent example of confusion it can cause. 

If the user expects that, say, `activemq.enabled` in `values.yaml` controls whether activemq is enabled, then they will also expect when `alfresco-infrastructure` is deployed as a dependency of a parent chart this value can be overriden through the master chart. For example, when pulled as part of `alfresco-dbp` chart, the user could reasonably assume that in that case to disable activemq they would need to override `alfresco-infrastructure.activemq.enabled` (because this is a child chart) when in reality this will make no effect on the condition in `requirements.yaml`. 

To me, this confusion is a bigger issue than users not going through documentation.

**Comment by rgauss** [comment]
> the user could reasonably assume that in that case to disable activemq they would need to override `alfresco-infrastructure.activemq.enabled` (because this is a child chart) when in reality this will make no effect on the condition in `requirements.yaml`.

Hmm, we seem to be inconsistent with our requirement conditions.  Should we remove the `alfresco-infrastructure` prefix from the requirement conditions in this chart?

**Comment by sergiuv2020** [comment]
No we shouldnt remove it. 

We always add the parent of the chart in the condition otherwise if a parent chart like dbp comes on top of infra along with another chart having an extra activemq then you would not be able to disable one of them. Just both since the conditions are global variables.

I do not see any inconsistency, we assign conditions like this everywhere mainly because we had a lot of issues with this before.

If we still want this here as an easy way to disable the components ill just write them the right way.

**Comment by rgauss** [comment]
> If we still want this here as an easy way to disable the components ill just write them the right way.

I think that would be helpful for newcomers.

For others' reference: https://github.com/helm/helm/issues/2479

**Review by rgauss — APPROVED** [review]

**Review by jottley — APPROVED** [review]

**Pull request merged**